docs/reference/cooperative_groups_reference.rst

```diff
@@ -0,0 +1,56 @@
+.. meta::
+  :description: This chapter lists new types and device API wrappers related to Cooperative Group
+                feature, which the programmer can directly use in his kernel(s) in order to
+                make use of this feature.
+  :keywords: AMD, ROCm, HIP, cooperative groups
+
+*******************************************************************************
+Cooperative Groups API
```

Comment:
Looks like this content is coming from elsewhere. Some comments.
Change this line everywhere: This function is implemented on Linux, under developement on Windows.
To: This function is implemented on Linux and is under development on Microsoft Windows.

Be gender neutral - see this example:
Instead, he should construct it through this API function.
It should be:
Instead, they should construct it through this API function.

Reply:
This is in clr. Updating the section too:

https://github.com/ROCm/clr/pull/81/commits

Reply:
Fixed.